### ISSUE TITLE
iOS  'admob.interstitial.dismiss' not fired

### ISSUE THREAD

**Issue by Sayene** [admob-plus/admob-plus#356]
**Describe the bug**

Error on iOS after Interstitial (any) is dismissed. No ads are shown after this error.

**To Reproduce**
1. create, load, and show InterstitialAd
```js
let interstitial

document.addEventListener('deviceready', async () => {
  await admob.start()

  interstitial = new admob.InterstitialAd({
    adUnitId: 'ca-app-pub-3940256099942544/4411468910'
  })
  await interstitial.load()
  await interstitial.show()
}, false)

document.addEventListener('admob.interstitial.dismiss', async () => {
  await interstitial.load()
})
```

2. dismiss the ad using ad's close button
3. check xcode console / log to see:
```
2021-05-28 18:09:23.505777+0200 APP.NAME[9013:3433669] [assertion] Error acquiring assertion: <Error Domain=RBSAssertionErrorDomain Code=3 "Target is not running or required target entitlement is missing" UserInfo={RBSAssertionAttribute=<RBSDomainAttribute| domain:"com.apple.webkit" name:"Suspended" sourceEnvironment:"(null)">, NSLocalizedFailureReason=Target is not running or required target entitlement is missing}>

2021-05-28 18:09:23.505945+0200 APP.NAME[9013:3433669] [ProcessSuspension] 0x106921cc0 - ProcessAssertion: Failed to acquire RBS Suspended assertion 'WebProcess Suspended Assertion' for process with PID 9036, error: Error Domain=RBSAssertionErrorDomain Code=3 "Target is not running or required target entitlement is missing" UserInfo={RBSAssertionAttribute=<RBSDomainAttribute| domain:"com.apple.webkit" name:"Suspended" sourceEnvironment:"(null)">, NSLocalizedFailureReason=Target is not running or required target entitlement is missing}
```
4. call to load done on admob.interstitial.dismiss event does not work (ad is not shown)

**Expected behavior**
no errors, ads load after previous interstitial were dismissed

**Environment**
```
OS: macOS Big Sur 11.3.1
CocoaPods: 1.10.1
Xcode: 12.5/12E262
iOS 14.5 (emulator) 
admob-plus-cordova: 1.12.4 => 1.12.4 
cordova: ^10.0.0 => 10.0.0 
cordova-ios: ^6.2.0 => 6.2.0 
```
**Additional context**

**Comment by Sayene** [comment]
looks similar to [capacitor-community/admob issue](https://github.com/capacitor-community/admob/issues/82), just fixed

**Comment by EMI-INDO** [comment]
@Sayene 

https://github.com/admob-plus/admob-plus/blob/master/examples/cordova/www/js/index.js

**Comment by Sayene** [comment]
@EMI-INDO , ok I've double checked the demo and ads do load after Interstitial was dismissed. However:

1.  'admob.interstitial.dismiss' is not fired (in my code I called other ads from this event - since they did not load)
2.  Target is not running ... error is logged - not sure what impact it has

On Android, the same code runs OK, 'admob.interstitial.dismiss' event is fired when interstitial add is closed.
I've updated to 1.13.0 but nothing changed there.
